SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/SpatialMultiWorkerSettings.cpp

```diff
@@ -27,13 +28,14 @@ void UAbstractSpatialMultiWorkerSettings::PostEditChangeProperty(struct FPropert
 		ValidateNoActorClassesDuplicatedAmongLayers();
 		ValidateAllLayersHaveUniqueNonemptyNames();
 		ValidateAllLayersHaveLoadBalancingStrategy();
+		ASpatialWorldSettings::EditorRefreshSpatialDebugger();
```

Comment:
This occurs when the layer info is changes, which as far as I knew, didn't have any visual impact on the spatial debugger. Why are we refreshing it here?

Reply:
The SpatialMultiWorkerSettings has a nested "Load Balance Strategy", if this changes AND if this is the current SpatialMultiWorkerSettings selected in the "World Settings" then this needs to update the visualisation. Similarly, if the "Worker Layer" is deleted this change will also need to be reflected.
I will add a check to see if this is the current SpatialMultiWorkerSettings before doing the refresh.
Similarly, I will add a check when the rows, cols etc change to check if the LoadBalancingStrategy is the current one before doing the refresh.

Reply:
Checks added.